blog/members/mangel-de6021aa.md

```
@@ -0,0 +1,53 @@
+        <a class="external reference" href="https://github.com/MangelRyujin">
+            <iconify-icon icon="simple-icons:github" style="font-size:2em"></iconify-icon>
+        </a>
+    </li>
+</ul>
+```
+
+:Aliases: Mangel
+
+:Ciudad: Lima, Perú
+
+:Homepage: https://mangel-portfolio.vercel.com
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)